recipes/spyboat/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "spyboat" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 87f1c8889b46b01e47c0ccb7be12cbca00a1bd31bc90271a746ad58ca8e573bd
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - flit-core >=2.3
+  run:
+    - python
```

Comment:
Would be good to specify some minimum bound on Python here and in `host` (e.g. `python >=3.6`) unless you really mean 2.7 is supported.

Reply:
Allright!